Training Modules/01 Introduction/calculator/solution.py

```diff
@@ -0,0 +1,8 @@
+n1 = int(input())
+n2 = int(input())
+
+print("Sum = {}".format(n1+n2))
+print("Subtraction = {}".format(n1-n2))
+print("Multiplication = {}".format(n1*n2))
+print("Division = {}".format(n1/n2))
+print("Exponential = {}".format(n1**n2))
```

Comment:
A similar example is already there in the training modules [here](https://github.com/Rishit-dagli/Python3-by-practice/blob/master/Training%20Modules/01%20Introduction/1.03%20Arithmetic%20Operators/solution.py). I don't see any changes to it, you have simply renamed the file.